lib/protocols.js

```diff
@@ -978,6 +1007,7 @@
     , ListenersMixin
     , NoListenerInterestsMixin
     , NoMediaTypesMixin
+    , Ping29Mixin
```

Comment:
I don't think we need to define decodePingResponse for protocol 22 and 25. 
We should keep the current behavior, check on the main branch: `decodeServerMediaTypes` is undefined for 22/25 (see noMediaTypesMixin), then a check it's performed in [decodeBody](https://github.com/infinispan/js-client/blob/f7065100ed32d3983d8e67ed5f06fafc95295dc6/lib/protocols.js#L353) and the decode process terminates if decoder is undefined